domains/jayz33.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jayz3314",
+        "email": "zachariahmed@hotmail.com"
+    },
+    "record": {
+        "CNAME": "jayz33.is-a.dev"
```

Comment:
invalid value

Reply:
What value can I add? Idk how to use cname